runtime-sdk/src/modules/rofl/policy.rs

```diff
@@ -40,6 +65,58 @@ pub enum AllowedEndorsement {
     /// Specific node can endorse the enclave.
     #[cbor(rename = "node")]
     Node(PublicKey),
+    /// Any node from a specific provider can endorse the enclave.
+    #[cbor(rename = "provider")]
+    Provider(Address),
+    /// Any provider instance where the given address is currently the admin.
+    #[cbor(rename = "provider_instance_admin")]
+    ProviderInstanceAdmin(Address),
```

Comment:
Can this be simplified to `Admin` or  `InstanceAdmin`, or are there other admins?